source/_posts/2015-06-17-mind-the-code.markdown

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Mind the Code"
+date: 2015-06-17 22:17:50 +0100
+comments: true
+author: Kimberley Cook
+author_email: kimberleycook91@gmail.com
+categories: london mind-the-code codebar
+---
+
+On Saturday 4th June we organised Mind the Code, a day dedicated to mindfulness. Our aim was to run a slightly different event to anything we have ever ran before, where the day consisted of talks, workshops and yoga.
+
+The day began with a few lightning talks from all of our sponsors and the first main talk of the day by Emily Atkinson ([@emilyatk](https://twitter.com/emilyatk)), developer and product manager at ([Moo](http://uk.moo.com). Emily’s talk focused on coding mindfully. The first half of her talk focused on viewing code as a means to solving a problem. She discussed that as a developer you do not sit down to ‘write code’, you sit down to solve a problem. The second half of her talk was trying to get the audience to think about how other people view your code, and compared this to the foundations of a house. You want a solid base so that other developers can jump into the project and build upon your code without ruining your previous work. 
+
+[![Mind the code - Emily Atkinson](/images/MTC-EmilyAtkinson.jpg)]()
+
+The second part to the day was an exercise on domain modelling. Students and coaches were paired up randomly and asked to design a new system for managing the London Tube system. Using paper and pens, we asked them to think about the concepts behind this model, classes, and who would be responsible for these classes and the relationship between them. 
+
+[![Mind the Code - Exercise 1](/images/MTC-Exercise1.jpg)]()
+
+Once the first half of the day was completed, the first Yoga session was taken by Sandrine. This was then followed by lunch, provided by [Pronto](http://pronto.co.uk).
+
+This was then followed by the start of our main coding workshop. The task here was to pick one of our set ([projects](https://github.com/deniseyu/mind-the-code/tree/master/projects) and begin building it with their coaches. The projects included a yoga slot machine, yoga lullabies, yoga pose name generator, online mode ring and a to-do list. The aim of this workshop was to get the students started on a projects of their choice.  
```

Comment:
Maybe change 'This was then followed' as is used in the line above to 'Next began our main coding workshop'